addons/ingress-citrix/README.md

```diff
@@ -30,6 +30,7 @@ For more information on how to enable addon during cluster creation refer [Kops
 After cluster creation, you can deploy [Citrix Ingress Controller](https://github.com/citrix/citrix-k8s-ingress-controller) in Google Cloud Platform using the below command
 
 ```
+kubectl create secret generic nslogin --from-literal=username='nsroot' --from-literal=password='nsroot'
```

Comment:
Perhaps have the example randomly generate the password?

Reply:
Taken care of it. Added a password placeholder.